tailwind.config.ts

```diff
@@ -28,6 +28,12 @@ export default {
         300: "#929292",
       },
     },
+    zIndex: {
+      layout: "1000",
+      object: "2000",
```

Comment:
What are `z-layout` and `z-object` used for?

Reply:
At the moment they're just placeholders by analogy to some of the named z-index values in Glides.